images/linux/scripts/installers/mysql.sh

```diff
@@ -44,7 +44,8 @@ echo "Lastly, documenting what we added to the metadata file"
 DocumentInstalledItem "MySQL ($(mysql --version))"
 DocumentInstalledItem "MySQL Server (user:root password:root)"
 DocumentInstalledItem "MS SQL Server Client Tools"
+DocumentInstalledItem "MySQL service is disabled by default. Use the following command as a part of your job to start the service: sudo /etc/init.d/mysql start"
```

Comment:
I think we should use systemctl to start the service:
`systemctl start mysql.service`